### DISCUSSION TITLE
day 15

### DISCUSSION THREAD

**Discussion by Junnyjun** [I-JUNNYLAND-I/algorithm#92] [Q&A] [answered]
[MANG JOO]

issue : #84 
pr : #85 
[leetcode](https://leetcode.com/problems/maximum-number-of-balls-in-a-box/)

**Comment by Junnyjun** [comment, marked as answer]
low 11, high 15 인 경우 11,12,13,14,15인 공을 특정 상자에 넣게됩니다

11인 경우 1+1 => Box 2
12인 경우 1+2 => Box 3
...

상자 중 가장 공이 많이 들어간 곳을 찾으면됩니다.

**Reply by Mang-Joo** [reply]
답변 감사합니다!
